lambda_functions/user_handler.py

```diff
@@ -0,0 +1,41 @@
+from crud import operations
+
+def delete_user(event, context, is_test=False, session=None):
```

Comment:
Is there a reason you didn't want to put this lambda function in app.py?

Reply:
Mainly to reduce the complexity in the app.py -  since the methods don't share any code I don't see a reason why they should be placed in the same file